ds/src/main/java/org/opendaylight/etcd/ds/impl/EtcdWatcher.java

```diff
@@ -76,36 +71,23 @@ public void close() {
     }
 
     private Watcher watch(long revision) {
-        Watcher watcher = etcdWatch.watch(prefix,
```

Comment:
may I ask for some more comments re. why you are removing all this?

Reply:
or perhaps you could revert this part of the change, just so that we get the build to work again, and then raise this a separate PR later?

Reply:
I now see why you (had to) do this.. this is an impact of https://github.com/etcd-io/jetcd/pull/419/ ..